service/service.go

```diff
@@ -33,6 +33,7 @@ type Service struct {
 	OutputQueue                searchoutputqueue.Output
 	SearchAPIURL               string
 	HierarchyBuiltProducer     *kafka.Producer
+	OTServiceName			   string
```

Comment:
Could the indentation be fixed here please so the "string" lines up with the "*kafka..." above.